python_modules/dagster/dagster/_core/definitions/time_window_partitions.py

```diff
@@ -488,6 +488,7 @@ def __repr__(self):
         # Replaces start time with timestamp as a workaround to make sure the repr is consistent across versions.
         return (
             f"TimeWindowPartitionsDefinition(start={self.start.timestamp()},"
+            f" end={self.end.timestamp() if self.end else None},"
```

Comment:
if we aren't going to be able to do a fix where we don't use `__repr__` for sorting soon, can you add a note to the attrs for `TimeWindowPartitionsDefinition` saying to update `__repr__` for any added attr?